encrypt.ts

```diff
@@ -0,0 +1,17 @@
+import { readFile, writeFile } from 'fs/promises'
+import { create } from '@chainsafe/bls-keystore'
+
+const PASSWORD = ''
```

Comment:
Maybe put it in cli args or env that we could use this example?

Reply:
And describe in readme how to run the command. I guess we will need to add in package.json in this case

Reply:
That's just a super simple example for now. No one will encrypt messages one by one, so it needs modifications anyway. But it's better than nothing. I propose to either leave it as-is or make a complete example with folder reading, pass in env, filename preservation and so on.

Reply:
Added a complete script, please take a look